src/js/apps/patients/patient/flow/flow_app.js

```diff
@@ -291,6 +293,10 @@ export default SubRouterApp.extend({
       },
     });
 
+    this.listenTo(userActivityCh, 'close:patient:action:sidebar', () => {
```

Comment:
`'close:patient:action:sidebar'` seems like a long string message. But wanted to make it as specific as possible since this channel is global throughout the app.

Reply:
I think we can do `close:patientActionSidebar`. or maybe `close:actionSidebar` since this for instance is a flow action

Reply:
Okay updated to `close:actionSidebar`